src/py42/_internal/clients/investigation.py

```diff
@@ -0,0 +1,40 @@
+from py42.clients import BaseClient
+
+
+class SavedSearchClient(BaseClient):
+    """A client to interact with saved search APIs."""
+
+    _version = u"v1"
+    _resource = u"/forensic-search/queryservice/api/{}/saved".format(_version)
+
+    _response_header = [u"name", u"id", u"notes", u"modifiedByUsername", u"modifiedTimestamp"]
+
+    def __init__(self, session, file_event_client):
+        super(SavedSearchClient, self).__init__(session)
+        self._file_event_client = file_event_client
+
+    def _parse_response(self, response):
+        if u"searches" in response:
+            return {
+                key: search[key]
+                for search in response[u"searches"]
+                for key in self._response_header
+            }
+
+    def get(self):
+        uri = u"{}".format(self._resource)
+        response = self._session.get(uri)
```

Comment:
Is there a reason you chose to not simply return `self._session.get(uri)` here?

Reply:
One of the acceptance criteria in the ticket is " **The output should be the same as if it were run in the console** "

Here I interpreted console as either of _web-console_ or _console.us.code42.com_ and hence set the fields that are displayed in web when a user wants to view existing saved search list and hence the parser in next line.

Reply:
Moreover didn't add the same to `get_by_id` so that if a user intends to view the complete saved search detail they can use the same, whereas `get` helps to view the id, name and other metadata.

Reply:
Also do note, used `get` over `get_all` because the response doesn't have paginated data. Since by convention `get_all` returns a generator to indicated paginated response.

Reply:
Sorry for the confusion -- since this api is not well documented, that instruction was meant to guide you to use the same call / response seen in the web console. Please have this simply return `self._session.get(uri)`.

Reply:
ok.